pkg/base/helm_v3.go

```diff
@@ -76,6 +82,29 @@ func renderHelmV3(chartName string, chartPath string, vals map[string]interface{
 		})
 	}
 
+	// this secret should only be generated for installs that rely on us rendering yaml internally - not native helm installs
+	// those generate their own secret
+	if !renderOptions.UseHelmInstall {
+		if renderOptions.Namespace == "" {
+			rel.Namespace = namespace()
+		}
+		rel.Info.Status = rspb.StatusDeployed
+
+		helmReleaseSecretObj, err := newSecretsObject(rel)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to generate helm secret")
+		}
+
+		renderedSecret, err := k8syaml.Marshal(helmReleaseSecretObj)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to generate helm secret")
+		}
+		baseFiles = append(baseFiles, BaseFile{
```

Comment:
This is going to show up in the diff for everybody upgrading to the latest version of KOTS, yes?  We should probably include a specific release note for this.

Reply:
I should also remove the timestamp (or make it a constant) so that there aren't diffs every update